src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -158,7 +158,9 @@ <h4 *ngIf="course.id" id="jhi-course-heading-edit" jhiTranslate="artemisApp.cour
                     ></fa-icon>
                 </div>
                 <div class="form-group" *ngIf="complaintsEnabled">
-                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaints.title" for="field_maxComplaints">Maximum Number of Complaints</label>
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaints.title" for="field_maxComplaints"
+                        >Maximum amount of complaints per student</label
```

Comment:
The formatting here looks a bit weird, can you maybe check this again?

Reply:
I fixed the formatting. I think that happens when prettier auto-wraps lines that are too long.